testdata/globalconfig.yml

```diff
@@ -1,9 +1,7 @@
 subject: "namespace:ns"
 revision: "2022"
 adapters:
-  - name: denychecker.1
-    kind: istio/denyChecker
-    impl: istio/denyChecker
+  - name: memQuota
```

Comment:
should we call this something other than memQuota to avoid potential confusion with the implementation?